components/renku_data_services/data_connectors/db.py

```diff
@@ -80,6 +80,25 @@ async def get_data_connector(
                 raise errors.MissingResourceError(message=not_found_msg)
             return data_connector.dump()
 
+    async def get_data_connectors_names(
```

Comment:
This is used to inform users which data connectors weren't copied. We display the name in the project page and we want to display the same information here.
Having the same name is allowed but it's not a good practice, so, in reality, it shouldn't happen. If it happens here, nothing functional-wise is wrong.

Reply:
I strongly disagree, because you will reply with a wrong count of Data Connectors which have an issue. The `namespace/slug` (identifier) is already something users manipulate as they have to use it to connect projects to existing data connectors. It is NOT upon the backend to generate nice error messages, but on the UI to take error data to render what is appropriate.

Reply:
That's something that we decided in the sync meeting and it's implemented according to our agreement. I don't see why it might cause an issue. I agree that it might cause some confusion but that's happening because the project owner used the same name for different data connectors in the first place.

Reply:
The project owner may not own the data connectors, so they may not have a choice here.

Reply:
Feel free to ignore, but we really need to end the bad practice of pushing error message rendering to the backend. This is bad practice and prevents anything like i18n from happening.

Reply:
I totally agree, but we decided to do it this way since UI's workload was high for this build.

Reply:
I've added `namespace/slug` to the error message in addition to the names.